docs/readmes/lte/deploy_install_docker.md

```diff
@@ -76,27 +74,10 @@ bash agw_install_docker.sh
 
 Once you see the output `Reboot this machine to apply kernel settings`, reboot your AGW host.
 
-After reboot again SSH and become root user:
-
-```bash
-sudo su
-cd /root
-```
-
-Now you have to update the environment file for docker-compose
-
-```bash
-sed -i 's,DOCKER_REGISTRY=,DOCKER_REGISTRY=docker.artifactory.magmacore.org/,' /var/opt/magma/docker/.env
-sed -i 's/IMAGE_VERSION=latest/IMAGE_VERSION=1.8.0/' /var/opt/magma/docker/.env
-
-# Update this only for ARM node
-sed -i 's/COMPOSE_PROJECT_NAME=agw/COMPOSE_PROJECT_NAME=agw_arm64/' /var/opt/magma/docker/.env
-```
-
 Create `control_proxy.yml` file with your orc8r details:
 
 ```bash
-cat << EOF > /var/opt/magma/configs/control_proxy.yml
+cat << EOF | sudo tee /var/opt/magma/configs/control_proxy.yml
```

Comment:
I think you need to add `sudo` in front of the following `docker-compose` and `docker` calls.